selfdrive/controls/lib/alerts_offroad.json

```diff
@@ -1,6 +1,6 @@
 {
   "Offroad_ChargeDisabled": {
-    "text": "EON charging disabled after car being off for more than 3 days. Turn ignition on to start charging again.",
+    "text": "EON charging disabled after car being off for more than 3 days or car voltage too low. Turn ignition on to start charging again or charge battery.",
```

Comment:
"or charge battery" <- let's remove this. As it is right now it won't switch back to CDP mode when voltage goes above 10.5V.

Reply:
so the only way to charge again is to turn on the car

Reply:
Shouldn't this be 'start engine to recharge battery' - as the ignition alone will only use more power.

Reply:
I see your point but it's kind of a corner case and currently we don't distinguish between ignition on VS engine started. Also, the EON must be fully operational with just ignition on, like any other ECU in the vehicle.